README.md

```diff
@@ -1,152 +1,335 @@
 # Minio Quickstart Guide
 [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-Minio is an object storage server released under Apache License v2.0. It is compatible with Amazon S3 cloud storage service. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images. Size of an object can range from a few KBs to a maximum of 5TB.
+Minio consists of an *object storage server*, released under Apache License v2.0, that is compatible with the Amazon S3 cloud storage service, as well as a *client component* used for sending and retrieving objects to and from the server. 
 
-Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL.
+Minio is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images. The size of an object can range from a few kilobytes to a maximum of 5TB. The Minio server component is light enough to be bundled with an application stack similar to NodeJS, Redis, and MySQL.
```

Comment:
The Minio server component is not quite right. Minio Server is not really a component per se. It's the actual server product.  What i mean is its not one component with server/client/sdk. Minio Server is the main product.

Reply:
@deekoder I have updated the branch with this change.